### DISCUSSION TITLE
My Glossary is not showing up in pdf; no error appears

### DISCUSSION THREAD

**Discussion by JOOO54965** [joaomlourenco/novathesis#467] [Q&A] [answered]
It is not showing in the TOC or the text itself, I'm using overleaf and the template version I'm not sure where to see, but it should be the latest. 

The Symbols and Abreviations are showing up but the Glossary even when I add new things and refer them in text is not showing.

Any help?

**Comment by joaomlourenco** [comment]
Can you please start a new project in overleaf (from the template) and confirm if the glossary shows or not?  It does to me…
If the glossary shows in the new project, it means it is something with your document.
Please double check:
* That the file "glossary.tex" is being included in ´Config/4_files.tex`.
* That your are referring at least one entry from the glossary using `\gls{key-to-glossary-word}`. 
* Check the PDF in the place you are referring the glossary entry and confirm it looks as expected.  What happens when you click in the glossary work? Normally it would jump to the glossary page.
* Open the file `template.log` and search for "Error" and "error".  Double check you have no errors.

**Reply by JOOO54965** [reply]
I checked all of it. It is there, when I press the link it doesn't go anywhere. It doesn't even give an indication that it is reference. Unlike the case for the symbols or abbreviations. I do have some errors I'm trying to fix. The one that I think could be causing it it's about the: "Package microtype Warning: I cannot find a spacing list for font `TS1/zplTLF/m/n/10.95' (context: `nonfrench'). Switching off spacing for this font on input line 228." It actually happens twice but the line in question is in template.tex file so it's hard to find where it is in the code.

**Comment by MariaHenriques** [comment]
Hello, I'm having the same problem with the Glossary not showing. When I click on the hiperlinked words it just sends me to the cover page. I tried creating a new overleaf project - the glossary appears - but when I upload my chapters the glossary doesn't appear anymore. 
Any Suggestions?

**Reply by joaomlourenco** [reply]
Please **BE SURE** you have no errors while processing your thesis (red box with a number besides the "compile" button).  If you do have errors, anything can happen and I can't help.

Do you have any errors?  If you do, fix them, then try again.  If the problem persists, then come back here.  Otherwise, just let me know here the problem was solved when the errors were fixed.

**Reply by MariaHenriques** [reply]
![WhatsApp Image 2023-10-07 at 17 24 33](https://github.com/joaomlourenco/novathesis/assets/87181093/78fcea47-be8d-41d0-9e9b-b02437115095)

**Reply by MariaHenriques** [reply]
These are the errors I have and don't understand how to fix them

**Reply by joaomlourenco** [reply, marked as answer]
These are not errors. These are warnings and (these ones) can be safely ignored.
Your problem must be elsewhere.

Go step by step. Start with a clean template, add your definitions for glossary to the existing glossary file, and then add the command
```latex
\glsaddal
```
to the very end of the glossary file.

Let me know if it printed your entries or not.

**Reply by MariaHenriques** [reply]
I started copying each chapter to the begining template. I was able to find that the error is somewhere at the end chapter 2, which is when the glossary maggically desapears. However, i can't identify the error.

**Reply by MariaHenriques** [reply]
I dont know what the error was, but it finally worked! Thank you for all the help.
